src/dataLoaders/components/collectorsWizard/CollectorsStepSwitcher.tsx

```diff
@@ -3,42 +3,29 @@ import React, {PureComponent} from 'react'
 
 // Components
 import SelectCollectorsStep from 'src/dataLoaders/components/collectorsWizard/select/SelectCollectorsStep'
-import SelectCollectorsStep2 from 'src/dataLoaders/components/collectorsWizard/select/SelectCollectorsStep2'
-import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 import PluginConfigSwitcher from 'src/dataLoaders/components/collectorsWizard/configure/PluginConfigSwitcher'
-import PluginCreateConfigurationWizard from 'src/dataLoaders/components/collectorsWizard/verify/PluginConfigSwitcher'
 
 import VerifyCollectorsStep from 'src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep'
 import {ErrorHandling} from 'src/shared/decorators/errors'
 
 // Types
 import {CollectorsStep} from 'src/types/dataLoaders'
-import {PluginCreateConfigurationStepProps} from 'src/writeData/components/PluginCreateConfigurationWizard'
+import {CollectorsStepProps} from 'src/dataLoaders/components/collectorsWizard/CollectorsWizard'
 
 interface Props {
-  // stepProps: CollectorsStepProps
-  stepProps: PluginCreateConfigurationStepProps
+  stepProps: CollectorsStepProps
 }
 
 @ErrorHandling
 class StepSwitcher extends PureComponent<Props> {
   public render() {
     const {stepProps} = this.props
-    let selector
-    let configure
-    if (isFlagEnabled('telegrafUiRefresh')) {
-      selector = <SelectCollectorsStep2 {...stepProps} />
-      configure = <PluginCreateConfigurationWizard />
-    } else {
-      selector = <SelectCollectorsStep {...stepProps} />
-      configure = <PluginConfigSwitcher />
-    }
 
     switch (stepProps.currentStepIndex) {
       case CollectorsStep.Select:
-        return selector
+        return <SelectCollectorsStep {...stepProps} />
```

Comment:
is this flagged elsewhere?

Reply:
No. It was flagged for one purpose and I moved this flagging to a higher component. The other work flow does not come into this hierarchy.